src/features/mod.rs

```diff
@@ -5,6 +5,7 @@ use std::{collections::HashSet, ffi::CStr};
 
 pub mod options;
 pub mod urid_map;
+pub mod worker;
```

Comment:
`iter_features` from this file also needs to include the worker feature for the plugin to not get filtered out.

Some solutions I see:
- Remove `iter_features` and inline all uses. I think its mostly for `supported_features` (in this file) and building the features when instantiating plugins.
- Have `iter_features` take in the worker `LV2Feature` as a param. For `supported_features`, you can pass in a constant that just has the URI with data set to `std::ptr::null`. See `BOUNDED_BLOCK_LENGTH` defined in this file.

Reply:
Thanks, I took the first approach. After thinking about it a bit I figured that the HashSet returned by `supported_features` and iterating over the features themselves are quite different use cases, so there's no need for `supported_features` to use on `iter_features`. I've simplified `supported_features` to just return `HashSet<&'static str>` since the contents of that I assume should not be expected to change at runtime?